examples/multi-tree.rs

```diff
@@ -0,0 +1,151 @@
+use indicatif::{MultiProgress, ProgressBar, ProgressStyle};
+use lazy_static::lazy_static;
+use std::sync::{Arc, Mutex};
+use std::thread;
+use std::time::Duration;
+
+use std::fmt::Debug;
+
+#[derive(Clone, Debug)]
+struct Rng(u64);
+
+impl Default for Rng {
+    fn default() -> Self {
+        Self(14312312512314u64)
```

Comment:
Huh?

Reply:
This is just to avoid `rand` dependency. You propose to get ms since epoch, right?